### PR TITLE
[web][SIM-168] Add option to search education program

### PR DESCRIPTION
**Pull request by fwpushan**
------------------------------------------------------
1. Add new EducationProgramList.vue component and integrate with SelectProgram page
2. Add some state management
3. Also added code for SIM - #169 

Screenshot 2021-02-20 at 3 14 27 AM

### CODE REVIEW

File: sources/packages/web/src/services/InstituteService.ts [hunk 1]
```diff
@@ -3,6 +3,11 @@ export interface Institute {
   code?: string;
 }
 
+export interface EducationProgram {
+  name: string;
+  code?: string;
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/services/InstituteService.ts:8]: Is there a reason to assume that this would be nullable?

**Review by andrewsignori-aot — APPROVED** [review]
👍

File: sources/packages/web/src/services/InstituteService.ts (hunk 1 above)
Comment (fwpushan) [reply sources/packages/web/src/services/InstituteService.ts:8]: For now this is placeholder only, we will revisit this after detailed API and data modelling.

**Review by 2011sandeepraj — APPROVED** [review]
LGTM

**Pull request merged**